modules/virt-adding-a-boot-source-web.adoc

```diff
@@ -35,17 +35,21 @@ There are four methods for selecting and adding a boot source in the web console
 
 . In the *Add boot source to template* window, click *Select boot source*, select a method for creating a persistent volume claim (PVC): *Upload local file*, *Import via URL*, *Clone existing PVC*, or *Import via Registry*.
 
-. Optional: Click *Mount this as a CD-ROM boot source* to mount a CD-ROM and use it to install the operating system on to an empty disk. The additional empty disk is automatically created and mounted by {VirtProductName}. If the additional disk is not needed, you can remove it when you create the virtual machine. 
+. Optional: Click *Mount this as a CD-ROM boot source* to mount a CD-ROM and use it to install the operating system on to an empty disk. The additional empty disk is automatically created and mounted by {VirtProductName}. If the additional disk is not needed, you can remove it when you create the virtual machine.
 
 . Enter a value for *Persistent Volume Claim size* to specify the PVC size that is adequate for the uncompressed image and any additional space that is required.
 
 .. Optional: Enter a name for *Source provider* to associate the name with this template.
 
-.. Advanced: Click *Storage class* and select the storage class that is used to create the disk.
+.. Optional: *Advanced Storage settings*: Click *Storage class* and select the storage class that is used to create the disk. Typically, this storage class is the default storage class that is created for use by for all PVCs.
```

Comment:
In the last sentence, the second "for" seems redundant. Change to "..created for use by all PVCs." or "..created for all PVCs."

Reply:
Updated the content.